Boards.h

```diff
@@ -348,6 +347,57 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_PWM(p)           PIN_TO_DIGITAL(p)
 #define PIN_TO_SERVO(p)         ((p) - 2)
 
+// pic32 board FubarinoSD
+#elif defined(_BOARD_FUBARINO_SD_)
+#define TOTAL_ANALOG_PINS 15
```

Comment:
Amazing timing. I just resynced my branch with the dev branch, oops. I'll
sort that out.

 I was going to redo a plain version without SoftPWMServo.  Then later do
something special for SoftPWMServo.

I'll line up the definitions better.

Also, I've been testing more closely and things are looking pretty good. So
I'll have a pull request soon.

-_Rick

On Sun, Feb 1, 2015 at 2:20 PM, Jeff Hoefs <notifications@github.com> wrote:

> In Boards.h
> <https://github.com/firmata/arduino/pull/154#discussion_r23900247>:
>
> > @@ -348,6 +347,57 @@ writePort(port, value, bitmask):  Write an 8 bit port.
> >  #define PIN_TO_PWM(p)           PIN_TO_DIGITAL(p)
> >  #define PIN_TO_SERVO(p)         ((p) - 2)
> >
> > +// pic32 board FubarinoSD
> > +#elif defined(_BOARD_FUBARINO_SD_)
> > +#define TOTAL_ANALOG_PINS 15
>
> formatting nit...please line up your values with the values in the other
> board definitions
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/firmata/arduino/pull/154/files#r23900247>.
>



-- 
Co-founder
Fair Use Building and Research (FUBAR) Labs
http://fubarlabs.org